dataactvalidator/config/sqlrules/b9_award_financial.sql

```diff
@@ -7,7 +7,7 @@ SELECT af.row_number,
 	af.program_activity_code
 FROM award_financial as af
 WHERE af.submission_id = {0}
-	AND CAST(COALESCE(SUBSTR(af.beginning_period_of_availa,1,4),'0') AS integer) >= 2016
```

Comment:
The substr was preventing it from catching errors for 5 digit years, which broke our current unit tests.  The B9 changes are in this PR because the errors in the unit tests were still coming from the old implementation, which was removed in this PR, so I needed to get the B9/B10 query to actually start catching some errors.